### DISCUSSION TITLE
AAAAAAAAAAAAAAAAAAAA

### DISCUSSION THREAD

**Discussion by ghost** [aflacc/ProjectFNF#70] [General]
THEY ADDED THE AFLAC MOD IN FUNKY FRICKIN FRIDAY

PLAY IT NOW

**Comment by ghost** [comment]
<img width="160" alt="AFLAC" src="https://user-images.githubusercontent.com/78268270/122248512-1bb20580-ce96-11eb-9ab8-836b4cc10714.PNG">

**Reply by PlayerPro342** [reply]
nice
